packages/coreutils/src/pageconfig.ts

```diff
@@ -154,6 +154,27 @@ export namespace PageConfig {
     return URLExt.normalize(wsUrl);
   }
 
+  /**
+   * Returns the URL converting this notebook to a certain
+   * format with nbconvert.
+   */
+  export function getNBConvertURL({
```

Comment:
@afshin Can you comment on how this would relate to your recent work with  `JupyterFrontEnd.IPaths` and `PageConfig`? What would be the preferred way for a notebook widget to get a handle on the NBConvert URL?

Reply:
Ping @afshin on this